apps/okreads-e2e/src/integration/reading-list.spec.ts

```diff
@@ -1,6 +1,8 @@
 describe('When: I use the reading list feature', () => {
+  let readingItemsCount = 0;
```

Comment:
you shouldn't be manually setting this value, what if the reading list already has items persisted from the API?